src/index.ts

```diff
@@ -422,6 +423,110 @@ const main = async () => {
     },
   );
 
+  tool(
+    'generate_dql_from_natural_language',
+    "Convert natural language queries to Dynatrace Query Language (DQL) using Davis CoPilot AI. DQL is the most powerful way to query any data in Dynatrace, including problem events, security issues, logs, metrics, spans, and custom data. This helps you write powerful DQL queries without knowing the exact syntax. Workflow: 1) Generate DQL, 2) Verify with verify_dql tool, 3) Execute with execute_dql tool, 4) Iterate if results don't match expectations. *(Note: Davis CoPilot AI is GA, but the Davis CoPilot APIs are in preview)*",
```

Comment:
I haven't reviewed the tool descriptions so far, but I want to be a bit nitpicky. Feel free to ignore me though, I just like to nitpick.

Tool descriptions are part of the context window, and verbose entries *may* confuse it. Perhaps we can find a more concise way to formulate this, without "informationless" phrases like "the most powerful way to query"

Reply:
I agree. I took this tool description from the other PR, and I'll refine it to be less verbose.